internal/locate/region_request3_test.go

```diff
@@ -36,6 +36,7 @@ package locate
 
 import (
 	"context"
+	"github.com/tikv/client-go/v2/metrics"
```

Comment:
Need to sort.

Reply:
It's updated in [my repo](https://github.com/you06/client-go/tree/reset-stale-read-6.5), but the display in PR is inconsistent.